etc/cas.properties

```diff
@@ -247,18 +256,18 @@ slo.callbacks.disabled=true
 #
 # Are multi-valued parameters accepted?
 # cas.http.allow.multivalue.params=false
-
+#
 # Define the list of request parameters to examine for sanity
 # cas.http.check.params=ticket,service,renew,gateway,warn,target,SAMLart,pgtUrl,pgt,pgtId,pgtIou,targetService
-
+#
 # Define the list of request parameters only allowed via POST
 cas.http.allow.post.params=password,oneTimePassword
 
 ##
 # JSON Service Registry
 #
 # Directory location where JSON service files may be found.
-service.registry.config.location=file:etc/services
+service.registry.config.location=file:etc/cas/services
```

Comment:
Need to revert this change back to `etc/services`.

* Settings in Helm Charts

```
service.registry.config.location=file:/etc/cas/services
log4j.config.location=file:/etc/cas/log4j2.xml
osf.api.institutions.auth.xslLocation=file:/etc/cas/institutions-auth.xsl
```

* Settings in CAS

```
service.registry.config.location=file:/etc/services
log4j.config.location=file:/etc/log4j2.xml
osf.api.institutions.auth.xslLocation=file:/etc/institutions-auth.xsl
```